core/src/main/java/edu/ucr/cs/riple/core/Config.java

```diff
@@ -387,8 +441,9 @@ private <T> OrElse<T> getValueFromKey(JSONObject json, String key, Class<T> klas
     }
   }
 
+  @SuppressWarnings({"SameParameterValue", "unchecked"})
```

Comment:
First one is IntelliJ warning that the input of this function is always and suggests to inline it, the other one is regarding calling map on stream of object. We expect JSONArray to return List of JSON Object here.

Reply:
The input value for now is always: `CONFIG_PATHS`. I think as this the tool evolves configurations can be more complex and this will be called with different values in future. Pleas let me know if you think I should inline this for now and update it future :)